sql/ksp/ksp.sql

```diff
@@ -203,27 +206,27 @@ CREATE FUNCTION pgr_ksp(
 RETURNS SETOF RECORD AS
 $BODY$
     SELECT *
-    FROM _pgr_ksp(_pgr_get_statement($1), _pgr_get_statement($2), $3, $4, $5, $6);
+    FROM  _v6pgr_ksp(_pgr_get_statement($1), _pgr_get_statement($2), $3, $4, $5, $6);
 $BODY$
 LANGUAGE SQL VOLATILE STRICT
 COST 100
 ROWS 1000;
 
 -- COMMENTS
 
-COMMENT ON FUNCTION pgr_ksp(TEXT, BIGINT, BIGINT, INTEGER, BOOLEAN, BOOLEAN)
-IS 'pgr_KSP(One to One)
-- Parameters:
-    - Edges SQL with columns: id, source, target, cost [,reverse_cost]
-    - From vertex identifier
-    - To vertex identifier
-    - K
-- Optional Parameters
-    - directed := true
-    - heap_paths := false
-- Documentation:
-    - ${PROJECT_DOC_LINK}/pgr_KSP.html
-';
+-- COMMENT ON FUNCTION pgr_ksp(TEXT, BIGINT, BIGINT, INTEGER, BOOLEAN, BOOLEAN)
```

Comment:
comment out the one-to-one that I created because one-to-one to is already there in older version